### ISSUE TITLE
Change "bing" to "bind"

### ISSUE THREAD

**Issue by kraftwerk15** [Equipple/vs-templates-revit-addin#2]
https://github.com/Equipple/vs-templates-revit-addin/blob/a5ed2bdebea294cf198c1e1d4e9392a5494f3deb/src/RibbonCommand.cs#L8

**Comment by salaros** [comment]
Fixed, thanks :+1: Please consider starring this repo
@kraftwerk15 would you like to have a slimmer version of this template (no ribbons etc)

**Comment by kraftwerk15** [comment]
@salaros I actually worked this a little in the opposite direction of starting to create WPF dialog boxes. But, this was a great start and loved not having to write a lot of the basics to get going.

I will add that having more explanation at [https://github.com/Equipple/vs-templates-revit-addin/blob/88873ddec0fa72c72d2652387222f06c480af0e6/src/RibbonHelper.cs#L225](url) would help on how to convert from a IExternalCommand to using it as a PushDataButton. That could be me not knowing the best way to convert it. I used [http://archi-lab.net/create-your-own-tab-and-buttons-in-revit/](url) when starting to learn and do not know all of the mechanisms to correlate between the two. Still in between the beginner and novice, but learning when possible.

Third, it would be helpful when adding on to 2020 to have a wiki in the repo explaining the changes that need to be made in the .csproj file and elsewhere to make Revit 2020 and .NET to work with the previous versions.
